sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/DistroWebAppLiveTests.cs

```diff
@@ -0,0 +1,138 @@
+
+        private LogsQueryClient? _logsQueryClient = null;
+
+        // DEVELOPER TIP: Can pass RecordedTestMode.Live into the base ctor to run this test with a live resource.
+        // DEVELOPER TIP: Can pass RecordedTestMode.Record into the base ctor to re-record the SessionRecords.
+        public DistroWebAppLiveTests(bool isAsync) : base(isAsync) { }
+
+        [RecordedTest]
+        public async Task VerifyDistro()
+        {
+            // SETUP LOGS CLIENT
+            _logsQueryClient = InstrumentClient(new LogsQueryClient(
+                TestEnvironment.LogsEndpoint,
+                TestEnvironment.Credential,
+                InstrumentClientOptions(new LogsQueryClientOptions()
+                {
+                    Diagnostics = { IsLoggingContentEnabled = true }
+                })
+            ));
+
+            _logsQueryClient.SetQueryWorkSpaceId(TestEnvironment.WorkspaceId);
+
+            // SETUP WEBAPPLICATION WITH OPENTELEMETRY
+            var resourceAttributes = new Dictionary<string, object>
+            {
+                { "service.name", _roleName },
+            };
+
+            var resourceBuilder = ResourceBuilder.CreateDefault();
+            resourceBuilder.AddAttributes(resourceAttributes);
+
+            var builder = WebApplication.CreateBuilder();
+            builder.Logging.ClearProviders();
+            builder.Services.AddOptions<OpenTelemetryLoggerOptions>()
+                .Configure(options =>
+                {
+                    options.SetResourceBuilder(resourceBuilder);
+                });
+            builder.Services.AddOpenTelemetry()
+                .ConfigureResource(x => x.AddAttributes(resourceAttributes))
+                .UseAzureMonitor(options =>
+                {
+                    options.ConnectionString = TestEnvironment.ConnectionString;
+                });
+
+            var app = builder.Build();
+            app.MapGet("/", () =>
+            {
+                app.Logger.LogInformation(_logMessage);
+
+                return "Response from Test Server";
+            });
+
+            _ = app.RunAsync(_testServerUrl);
+
+            // ACT
+            using var httpClient = new HttpClient();
+            var res = await httpClient.GetStringAsync(_testServerUrl).ConfigureAwait(false);
+            Assert.True(res.Equals("Response from Test Server"), "If this assert fails, the in-process test server is not running.");
+
+            // SHUTDOWN
+
+            // NOTE: If this test starts failing, Flushing may be necessary.
+            //var tracerProvider = app.Services.GetRequiredService<TracerProvider>();
+            //tracerProvider.ForceFlush();
+            //tracerProvider.Shutdown();
+
+            await app.StopAsync(); // shutdown to prevent collecting the log queries.
+
+            // ASSERT
+            // TODO: NEED TO PERFORM COLUMN LEVEL VALIDATIONS.
+            await VerifyLogs(
```

Comment:
probably good to add why we are querying `AppDependencies` and not `Dependencies` as a note

Reply:
Are you looking for a note that we're querying using the LogsAnalytics schema, or something else?

Reply:
Yes - that looks good